Author: Dave Jones <dave.jones@canonical.com>
Description: Use /usr/bin/python3 explicitly in rpi-eeprom-config
 The python symlink no longer exists by default so default configurations must
 explicitly specify a python binary version

--- a/rpi-eeprom-config
+++ b/rpi-eeprom-config
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 # rpi-eeprom-config
 # Utility for reading and writing the configuration file in the
